internal/host/installcmd.go

```diff
@@ -91,9 +91,12 @@ func (i *installCmd) GetStep(ctx context.Context, host *models.Host) (*models.St
 			data["HTTPS_PROXY"] = cluster.HTTPSProxy
 		}
 		if cluster.NoProxy != "" {
-			cmdArgsTmpl = cmdArgsTmpl + " --no-proxy {{.NO_PROXY}}"
-			data["NO_PROXY"] = cluster.NoProxy
+			data["NO_PROXY"] = cluster.NoProxy + ","
 		}
+		cmdArgsTmpl = cmdArgsTmpl + " --no-proxy {{.NO_PROXY}}"
+		// if we set proxy we need to update assisted installer no proxy with api int value
+		// it must be able to connect to api int. Added this way for not to pass name and base domain
+		data["NO_PROXY"] = data["NO_PROXY"] + fmt.Sprintf("api-int.%s.%s", cluster.Name, cluster.BaseDNSDomain)
```

Comment:
Let's include all of the static values as set by default by openshift-install as listed on 
https://github.com/openshift/installer/blob/master/pkg/asset/manifests/proxy.go#L125

```
	set := sets.NewString(
		"127.0.0.1",
		"localhost",
		".svc",
		".cluster.local",
		internalAPIServer.Hostname(),
	)
```